src/components/organizations/credentials.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { organizationCredentialsLoad } from '../../actions/organizationActions';
+
+class Credentials extends React.Component {
+  componentDidMount() {
+    this.props.dispatch(organizationCredentialsLoad(this.props.organizationName));
+  }
+
+  render() {
+    var provider = this.props.app.info.general.provider;
+
+    if (this.props.credentials.isFetching) {
+      return (
+        <span>
+          <img className='loader' src='/images/loader_oval_light.svg' width='20px' height='20px' /> <span>Loading credentials</span>
+        </span>
+      );
+    } else {
+      if (this.props.credentials.items.length === 0) {
+
+        var gsctlInfo = <p>You can set credentials to use your own subscription using <a href='https://docs.giantswarm.io/reference/gsctl/update-org-set-credentials/' target='_blank' rel='noopener noreferrer'>our CLI</a></p>;
+
+        if (provider === 'azure') {
+          return (
+            <div>
+              <p>No credentials set. Clusters of this organization will be created in the default tenant cluster subscription configured for this installation.</p>
```

Comment:
> No credentials set.
This reads a bit odd/worrying to me. There are _some_ credentials set (the default ones).

Maybe something more like 'No additional credentials available'?

Reply:
Interesting. The last thing I want is to worry non-BYOC users that something is wrong. How about "No specific provider credentials set. (...)"?

Reply:
I think that's an improvement. We can iterate with user feedback if needed.